### ISSUE TITLE
Parameter with the same name in different location of an operation is valid

### ISSUE THREAD

**Issue by mbohlool** [go-openapi/validate#7]
For example, for a path such as /api/{param}/op?param=... that has a param in "path" and a param in "query", the validation will fail while this is a valid schema.

**Comment by casualjim** [comment]
duplicate of #3 ?

**Comment by mbohlool** [comment]
Yes, and the location to fix it is in analysis repo. I am closing this in favor of go-openapi/validate#6. Will send a pull request shortly.
